.github/workflows/ssh_session.yml

```diff
@@ -26,39 +49,55 @@ jobs:
               --substituters 'https://cache.nixos.org https://cachix.cachix.org' \
               --trusted-public-keys 'cachix.cachix.org-1:eWNHQldwUO7G2VkjpnjDbWwy4KQ/HNxht7H4SSoMckM= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY='
 
+      - name: Create /var/tmp
+        run: |
+          mkdir -p /var/tmp
```

Comment:
Why `/var/tmp` instead of `/tmp`? `/tmp` gets automatically cleaned on reboot, does that make a difference?